### ISSUE TITLE
Extra newline in empty attribute

### ISSUE THREAD

**Issue by Nainoia-Inc-Admin** [speedata/publisher#306]
Moving from version 4.1.25 to 4.3.1 resulted in significant newline changes in some of my layout.  I need help understanding how to correct my layout XML.  The layout and two version examples are attached below.

[LayoutXMLSnippet.txt](https://github.com/speedata/publisher/files/5825529/LayoutXMLSnippet.txt)

[Example as preferred with version 4.1.25.pdf](https://github.com/speedata/publisher/files/5825521/Example.as.preferred.with.version.4.1.25.pdf)

[Example with extra space with version 4.3.1.pdf](https://github.com/speedata/publisher/files/5825522/Example.with.extra.space.with.version.4.3.1.pdf)

**Comment by pgundlach** [comment]
Looks like a bug, I'll investigate

**Comment by pgundlach** [comment]
Version 4.3.1


```xml
<data>
    <elt att1="a" att2="" />
    <elt att1="a" att2="" />
    <elt att1="a" att2="" />
</data>
```

```xml
<Layout xmlns="urn:speedata.de:2009/publisher/en"
    xmlns:sd="urn:speedata:2009/publisher/functions/en">

    <Record element="data">
        <PlaceObject>
            <Textblock>
                <Paragraph>
                  <ForAll select="elt">
                    <Value select="@att1"/>
                    <Value select="@att2"/>
                  </ForAll>
                </Paragraph>
            </Textblock>
        </PlaceObject>
    </Record>
</Layout>
```

Should render as `aaa` in one line.

**Comment by pgundlach** [comment]
This is fixed in version 4.3.2
